<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QPushButton" name="calbut">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcular</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>190</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="resposta_edit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>330</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>340</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>430</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>460</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_10">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>490</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x_11">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>520</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>280</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>340</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>370</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_8">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>430</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_9">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>460</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>490</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_y_11">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>520</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Metodo De Newton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_estipulado">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>260</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ponto de X Requisitado</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
